esp/src/eclwatch/templates/HPCCPlatformFilesWidget.html

```diff
@@ -6,12 +6,12 @@
         <div id="${id}TabContainer" data-dojo-props="region: 'center', tabPosition: 'top'" style="width: 100%; height: 100%" data-dojo-type="dijit.layout.StackContainer">
             <div id="${id}_LogicalFiles" title="${i18n.LogicalFiles}" data-dojo-props="delayWidget: 'DFUQueryWidget'" data-dojo-type="DelayLoadWidget">
             </div>
-            <div id="${id}_LandingZones" title="${i18n.LandingZones}" data-dojo-props="delayWidget: 'LZBrowseWidget'" data-dojo-type="DelayLoadWidget">
+            <div id="${id}_LandingZones" title="${i18n.LandingZones}" data-dojo-props="delayWidget: 'LZBrowseWidget', disabled:${dojoConfig.isContainer}" data-dojo-type="DelayLoadWidget">
```

Comment:
Is this browsing files on a landing zone?  I think this still makes sense (although isn't currently working for azureblobs).

Reply:
I can leave it enabled if the expectation is that we will have Azure landing zone support for v8?

Reply:
I am planning to implement it in the next few weeks.